src/urllib3/util/retry.py

```diff
@@ -95,6 +95,15 @@ class Retry(object):
 
         Set to ``0`` to fail on the first retry of this type.
 
+    :param int other:
+        How many times to retry on other errors.
+
+        Other errors are errors that are not connect, read, redirect or status errors.
+        These errors might be raised after the request was sent to the server, so the
+        request might have side-effects.
+
```

Comment:
For `total`, we say:

> It's a good idea to set this to some sensibly-high value to account for unexpected edge cases and avoid infinite retry loops.

However, with `other`, I believe the situation changes. I'd remove this line from total and add this line here:

> It's a good idea to set this to 0 to account for unexpected edge cases and avoid infinite retry loops.

What do you think?

Reply:
You are right :)
I've added the prefix:
> If ``total`` is not set,